src/platform.hpp

```
@@ -44,7 +44,7 @@ typedef std::optional<ResolvedCall> (*ebpf_resolve_kfunc_call_fn)(int32_t btf_id
 // In the future we may want to move map fd allocation after the verifier step.
 typedef void (*ebpf_parse_maps_section_fn)(std::vector<EbpfMapDescriptor>& map_descriptors, const char* data,
```

Clang (14.0.6)

[warning] use 'using' instead of 'typedef'

(modernize-use-using)